### PR TITLE
Issue#28 - create `Color` extension and `AppScheme` enum

### PR DESCRIPTION
**Pull request by marinkman**
## What?

- Created `Color` extension
- Created `AppScheme` enum and filled with `static let` colors according to the [Project color scheme](https://github.com/ios-course/swiftcowboys-team-project/wiki/Project-color-scheme)
- Created `AppScheme` class to tests `AppScheme` enum

## Why?
According to issue #28  

## Anything Else?
Need to discuss folder/file structure in test targets.

### CODE REVIEW

**Comment by nek-to** [comment]
I like your code. Honestly, I started the implementation the same way you did. I have no questions about the documentation and the tests. They may not be complete but they look good to me. The only thing is that in DoD we are told to create an Enum of Colors and you have an Extension created. I think if everyone agrees, you could tweak it.

That's the only blocking factor for LGTM from me.

File: Sleep Tracker/Sleep Tracker/Helpers/Color+Extension.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+extension Color {
+    /// A custom color representation that adapts to the given context.
```
[the remark this answers is not in the document]
Comment (MikhailKosarev) [reply Sleep Tracker/Sleep Tracker/Helpers/Color+Extension.swift:4]: According to the use of this extension "AppScheme" seems most appropriate - e.g. `Color.AppScheme.darkIndigo`

**Comment by marinkman** [comment]
@MikhailKosarev Why don't you support this idea?

**Comment by marinkman** [comment]
@MikhailKosarev It doesn't take long. I've already done it.

File: Sleep Tracker/Sleep Tracker/Helpers/Color+Extension.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import SwiftUI
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Sleep Tracker/Sleep Tracker/Helpers/Color+Extension.swift:1]: That's really better.
Done.

File: Sleep Tracker/Sleep TrackerUITests/CustomColorTests.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+import XCTest
+
+final class CustomColorTests: XCTestCase {
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Sleep Tracker/Sleep TrackerUITests/CustomColorTests.swift:4]: Yes, it is needed. Test added.

File: Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        XCTAssertEqual(white, Color.AppScheme.white)
+        XCTAssertEqual(black, Color.AppScheme.black)
+    }
+
+    func testColorAppSchemeCanBeUsedInAFunctionWithAColorParameter() {
+        func isTheColorUsed(color _: Color) -> Bool {
+            true
+        }
+
+        XCTAssertTrue(isTheColorUsed(color: Color.AppScheme.darkIndigo))
```
Comment (MikhailKosarev) [inline Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift:34]: just for info, we can you this way:
```suggestion
        XCTAssertTrue(isTheColorUsed(color: .AppScheme.darkIndigo))
```

**Review by MikhailKosarev — APPROVED** [review]
LGTM

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Please consider one more little thing:

File: Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift (hunk 4 above)
Comment (marinkman) [reply Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift]: Yes, exactly, thanks.
Done it.

File: Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import SwiftUI
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Sleep Tracker/Sleep TrackerUITests/AppSchemeTests.swift]: Yes, thanks. 
I created a `Helpers` folder in the test target and renamed the file `AppSchemeTeests` to `Color+AppSchemeTests`.

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**